zeppelin-client-examples/src/main/java/org/apache/zeppelin/client/examples/ZeppelinClientExample.java

```diff
@@ -41,6 +41,10 @@ public static void main(String[] args) throws Exception {
       noteId = zClient.createNote(notePath);
       System.out.println("Created note: " + noteId);
 
+      zClient.renameNote(noteId, notePath + "_rename");
+      NoteResult renamedNoteResult = zClient.queryNoteResult(noteId);
+      System.out.println("Rename note: " + noteId + " name to " + renamedNoteResult.getNoteName());
```

Comment:
Use assert to verify the new note name is the expected value. Don't use `System.out.println` in unit test, it could not verify the correctness

Reply:
That's certainly right, and I used assert now

Reply:
@zjffdu You are sure that you want to use the assert function of JUnit outside of a test class?